Pds4File/pds4_rules/cassini_vims.py

```diff
@@ -0,0 +1,763 @@
+])
+
+####################################################################################################################################
+# OPUS_FORMAT
+####################################################################################################################################
+
+opus_format = translator.TranslatorByRegex([
+    (r'.*\.IMG',        0, ('Binary', 'VICAR')),
+    (r'.*\.jpeg_small', 0, ('Binary', 'JPEG')),
+])
+
+####################################################################################################################################
+# OPUS_PRODUCTS
+####################################################################################################################################
+
+opus_products = translator.TranslatorByRegex([
+    (r'.*/(COISS_[12]xxx)(|_v[0-9\.]+)/(COISS_[12]...)/data/(\w+/[NW][0-9]{10}_[0-9]+).*', 0,
+            [r'volumes/\1*/\3/data/\4.IMG',
+             r'volumes/\1*/\3/data/\4.LBL',
+             r'volumes/\1*/\3/extras/thumbnail/\4.IMG.jpeg_small',
+             r'volumes/\1*/\3/extras/browse/\4.IMG.jpeg',
+             r'volumes/\1*/\3/extras/full/\4.IMG.png',
+             r'volumes/\1*/\3/extras/tiff/\4.IMG.tiff',
+             r'calibrated/\1*/\3/data/\4_CALIB.IMG',
+             r'calibrated/\1*/\3/data/\4_CALIB.LBL',
+             r'previews/\1/\3/data/\4_full.png',
+             r'previews/\1/\3/data/\4_med.jpg',
+             r'previews/\1/\3/data/\4_small.jpg',
+             r'previews/\1/\3/data/\4_thumb.jpg',
+             r'metadata/\1/\3/\3_moon_summary.tab',
+             r'metadata/\1/\3/\3_moon_summary.lbl',
+             r'metadata/\1/\3/\3_ring_summary.tab',
+             r'metadata/\1/\3/\3_ring_summary.lbl',
+             r'metadata/\1/\3/\3_saturn_summary.tab',
+             r'metadata/\1/\3/\3_saturn_summary.lbl',
+             r'metadata/\1/\3/\3_jupiter_summary.tab',
+             r'metadata/\1/\3/\3_jupiter_summary.lbl',
+             r'metadata/\1/\3/\3_inventory.csv',
+             r'metadata/\1/\3/\3_inventory.lbl',
+             r'metadata/\1/\3/\3_index.tab',
+             r'metadata/\1/\3/\3_index.lbl',
+             r'documents/COISS_0xxx/*.[!lz]*',
+            ]),
+])
+
+####################################################################################################################################
+# OPUS_ID
+####################################################################################################################################
+
+#opus_id = translator.TranslatorByRegex([
+#    (r'.*/COISS_[12]xxx.*/([NW][0-9]{10})_[0-9]+.*', 0, r'co-iss-#LOWER#\1'),
+#])
+
+
+opus_id = translator.TranslatorByRegex([
+    (r'.*/cassini_vims/cassini_vims\w*/data_raw/\d{3}xxxxxxx/\d{5}xxxxx/\d{10}_xxx/(\d{10}_\d{3}).[a-z]{3}|.*/cassini_vims/cassini_vims\w*/data_raw/\d{3}xxxxxxx/\d{5}xxxxx/(\d{10}).[a-z]{3}', 0, r'co-vims-v\1\2_vis'),
+]) # Hard-code in "_vis" for the moment, but could be "_ir". Sort this out later. 
```

Comment:
So here we don't want to hard code and append  `_vis` or `_ir`, we will map to something like `r'co-vims-v\1\2'`
Here is an example of an pds3 file instance:
```
In [5]: a3 = pdsfile.PdsFile.from_abspath('/Users/yu-jenchang/Dropbox (SETI Institute)/Shared-OPUS/pdsdata/holdings/vo
   ...: lumes/COVIMS_0xxx/COVIMS_0001/data/1999017T031657_1999175T202056/v1308946681_1_002.qub')

In [6]: a3.opus_id
Out[6]: 'co-vims-v1308946681_002'
```
As you can see from above results, the appended `_vis` and `_ir` will be handled somewhere else, here we just ignore the them.

Reply:
I see, I've removed the placeholder `_vis` from the COVIMS opus_id